package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@livingdocs/node-sdk",
```

Comment:
The `name` is outdated and points to the old repo. Can you fix this in this PR as well?

Reply:
Thanks @mokimo for pointing this out 👍

Reply:
I am a bit confused, how should it be named instead? see https://www.npmjs.com/package/@livingdocs/node-sdk

Reply:
Or shall we rename it entirely and republish it with another name?

Reply:
Commented on the outdated "homepage", which you already fixed further down - I think Lukas referred to that!

Reply:
No, no. I commented on this. But I was wrong, you two were right ;)